### PR TITLE
FujitsuClientComputingLimited.DeskUpdate version 5.2

### PR DESCRIPTION
**Pull request by TargetCrafter**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/65255)

### CODE REVIEW

File: manifests/f/FujitsuClientComputingLimited/DeskUpdate/5.2/FujitsuClientComputingLimited.DeskUpdate.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: FujitsuClientComputingLimited.DeskUpdate
+PackageVersion: 5.2
```
Comment (ItzLevvie) [inline manifests/f/FujitsuClientComputingLimited/DeskUpdate/5.2/FujitsuClientComputingLimited.DeskUpdate.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 5.2.40.0
```
This must match Control Panel.

File: manifests/f/FujitsuClientComputingLimited/DeskUpdate/5.2/FujitsuClientComputingLimited.DeskUpdate.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: FujitsuClientComputingLimited.DeskUpdate
+PackageVersion: 5.2
```
Comment (ItzLevvie) [inline manifests/f/FujitsuClientComputingLimited/DeskUpdate/5.2/FujitsuClientComputingLimited.DeskUpdate.installer.yaml:5]:
```suggestion
PackageVersion: 5.2.40.0
```
This must match Control Panel.

File: manifests/f/FujitsuClientComputingLimited/DeskUpdate/5.2/FujitsuClientComputingLimited.DeskUpdate.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: FujitsuClientComputingLimited.DeskUpdate
+PackageVersion: 5.2
```
Comment (ItzLevvie) [inline manifests/f/FujitsuClientComputingLimited/DeskUpdate/5.2/FujitsuClientComputingLimited.DeskUpdate.yaml:5]:
```suggestion
PackageVersion: 5.2.40.0
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by TargetCrafter** [comment]
@ItzLevvie
I applied the suggested changes, could you approve the pr?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You need to accept the suggestions so it can make a commit.

You'll also need to update the version folder to match PackageVersion.

**Review by TargetCrafter — COMMENTED** [review]
Applied requested changes

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
I'm on my phone but you need to change the version to `5.2.40.0` to match Control Panel.

There should be a button called "Add suggestions to batch" (when you click on "Show resolved") and then make a commit and then you will need to update the version folder.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You'll also need to update the version folder to `5.2.40.0` to match `PackageVersion: 5.2.40.0` (from my suggestions) or the pipelines will error out.

**Comment by TargetCrafter** [comment]
Ended up cloning the repo to my pc to make the changes.
All should be good now.

**Review by ItzLevvie — APPROVED** [review]

**Comment by TargetCrafter** [comment]
Thanks for the help @ItzLevvie Ill delete the branch now

**Pull request merged**